core/src/main/java/uk/ac/bbsrc/tgac/miso/core/manager/ERASubmissionManager.java

```diff
@@ -494,15 +795,7 @@ public UploadReport getUploadProgress(Long submissionId) {
     return null;
   }
 
-  @Override
-  public void setTransferMethod(TransferMethod transferMethod) {
-
-  }
-
-  /*
-   * public FTPUploadReport getUploadReport(Submission submission){ return uploadReports.get(submission); }
-   */
-
+  // TODO(apmasell): These files no longer get written to disk, so this method doesn't work.
```

Comment:
As long as it's in the scope of "before we merge into develop," in which case more tickets please :tickets: :tickets: :tickets: :tickets: :tickets: ...